config/routes.rb

```diff
@@ -37,6 +37,7 @@
   post "/documents/:id/create-preview" => "preview#create", as: :create_preview
 
   get "/documents/:id/withdraw" => "withdraw#create", as: :withdraw
+  post "/documents/:id/withdraw" => "withdraw#new", as: :new_withdrawal
```

Comment:
May be being pedantic but shouldn't these two routes have each others action. e.g the get request for the form should be linked to the `new` action and the `post` request should be linked to the `create` action in the withdraw controller.

Reply:
That Rails book is coming in handy! Good catch 👍

Reply:
It looks like this didn't actually get resolved? 

Also I don't think we need the `as: :new_withdrawal` as we could use `withdraw_path(@document)` in the place we call that as it's the same path.